terraform/aws/module-ec2/variables.tf

```diff
@@ -0,0 +1,168 @@
+# Cycloid requirements
+variable "env" {
+  description = "Cycloid project name."
+}
+
+variable "project" {
+  description = "Cycloid environment name."
+}
+
+variable "customer" {
+  description = "Cycloid customer name."
+}
+
+###
+# AMI DATA
+###
+variable "ami_most_recent"{
+    description = "If more than one result is returned, use the most recent AMI."
+    default = true
+}
+
+variable "ami_name"{
+    description = "The name of the AMI (provided during image creation)."
+    default = "debian-stretch-*"
+}
+
+variable "ami_virtualisation_type"{
+    description = "The AMI virtualization type."
+    default = "hvm"
+}
+
+variable "ami_architecture"{
+    description = "The AMI image architecture"
+    default = "x86_64"
+}
+
+variable "ami_root_device_type"{
+    description = "The AMI type of the root device volume."
+    default = "ebs"
+}
+
+variable "ami_owners"{
+    description = "List of AMI owners to limit search."
+    default = "379101102735"
+}
+
+###
+# Cloud init template
+###
+variable "template_file"{
+    description = "The path and file name of the Cloud-init file to launch startup configuration in the vm."
+}
+
+###
+# Security Group
+###
+variable "sg_name"{
+    description = "Name of the security group."
+    default = ""
+}
+
+variable "vpc_id"{
+    description = "VPC ID used to create the security group."
+    default = ""
+}
+
+variable "sg_ingress_rules"{
+    description = "Configuration block for ingress rules."
+    default     = []
```

Comment:
As discussed maybe provide a default value here and in stackforms to open 22 port